iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/iothub/messaging/SendMessagesAndroidRunner.java

```diff
@@ -26,19 +23,15 @@
 import java.net.URISyntaxException;
 import java.security.GeneralSecurityException;
 import java.util.Collection;
-import java.util.Collections;
 
-@TestGroup33
+@TestGroup9
 @RunWith(Parameterized.class)
-public class SendMessagesModuleAndroidRunner extends SendMessagesTests
+public class SendMessagesAndroidRunner extends SendMessagesTests
 {
     @Rule
     public Rerun count = new Rerun(3);
 
-    @Rule
-    public ReportHelper reportHelper = Factory.getReportHelper();
-
-    public SendMessagesModuleAndroidRunner(IotHubClientProtocol protocol, AuthenticationType authenticationType, ClientType clientType, String publicKeyCert, String privateKey, String x509Thumbprint, boolean useHttpProxy) throws Exception
+    public SendMessagesAndroidRunner(IotHubClientProtocol protocol, AuthenticationType authenticationType, ClientType clientType, String publicKeyCert, String privateKey, String x509Thumbprint, boolean useHttpProxy) throws Exception
```

Comment:
nit: It would help if these constructors and parameterized test methods have some comments that describe what they do. I know this has been a point of confusion for me in the past, because these tests have pretty deep inheritance, and the test class and method names are not descriptive of what they are responsible for.

Reply:
Sure thing, I can add some comments. I plan on removing the JVMRunner classes and moving their logic into the test classes themselves to help cut down on the inheritance.

Reply:
This effort will be in a different PR, since removing the JVMRunner classes is a big task